client/grpc/client.go

```diff
@@ -7,6 +7,8 @@ import (
 	"fmt"
 	"time"
 
+	"github.com/drand/drand/protobuf/common"
```

Comment:
why is this in a separate import block and not next to `protobuf/drand` below?

Reply:
Fixed